manifests/params.pp

```diff
@@ -11,6 +11,27 @@
     fail('This module only works on Debian or derivatives like Ubuntu')
   }
 
+  $config_files = {
+    'conf'   => {
+      'path' => $conf_d,
+      'ext'  => '',
+    },
+    'pref'   => {
+      'path' => $preferences_d,
+      'ext'  => '',
+    },
+    'list'   => {
+      'path' => $sources_list_d,
+      'ext'  => '.list',
+    }
+  }
+
+  $file_defaults = {
+    'owner' => 0,
+    'group' => 0,
```

Comment:
![](http://i.giphy.com/WJK2SABYwvEvm.gif)

I think 'root' is more consistent over puppetlabs modules, so lets do that.